src/planwise/client/sources.cljs

```diff
@@ -8,41 +8,41 @@
 
 (def in-sources (rf/path [:sources]))
 
-(def initial-db
-  {:list (asdf/new nil)})
+;(def initial-db
+;  {:list (asdf/new nil)})
 
 ;; ----------------------------------------------------------------------------
 ;; API methods
 
-(def load-sources
-  {:method    :get
-   :section   :show
-   :uri       "/api/sources"})
+;(def load-sources
+;  {:method    :get
+;   :section   :show
+;   :uri       "/api/sources"})
 
 ;; ----------------------------------------------------------------------------
 ;; Listing sources
 
-(rf/reg-event-fx
- :sources/load
- in-sources
- (fn [{:keys [db]} [_]]
-   {:api (assoc load-sources
-                :on-success [:sources/loaded])
-    :db  (update db :list asdf/reload!)}))
-
-(rf/reg-event-db
- :sources/loaded
- in-sources
- (fn [db [_ sources]]
-   (update db :list asdf/reset! sources)))
+;(rf/reg-event-fx
+; :sources/load
+; in-sources
+; (fn [{:keys [db]} [_]]
+;   {:api (assoc load-sources
+;                :on-success [:sources/loaded])
+;    :db  (update db :list asdf/reload!)}))
+;
+;(rf/reg-event-db
+; :sources/loaded
+; in-sources
+; (fn [db [_ sources]]
+;   (update db :list asdf/reset! sources)))
 
 ;; ----------------------------------------------------------------------------
 ;; Subs
 
-(rf/reg-sub
- :sources/list
- (fn [db _]
-   (get-in db [:sources :list])))
+;(rf/reg-sub
+; :sources/list
+; (fn [db _]
+;   (get-in db [:sources :list])))
 
 (rf/reg-sub
  :sources/dropdown-options
```

Comment:
You may want to move the rest of the functions in this file to the corresponding namespaces.